.github/stale.yml

```diff
@@ -0,0 +1,55 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Label to use when marking as stale
+staleLabel: Stale
+
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+# limitPerRun: 30
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
That seems very long. I would prefer a shorter time here and a longer below (`daysUntilClose`). I often see marking as stale as an incentive for contributors to spend some time on the issue/PR. Plus, after 2 months, it's hard for anyone involved to remember what the issue was.

Reply:
How about 30 / 14? We can also use different time frames for issues and PRs. What would you prefer?

Reply:
30/14 looks good to me :+1: 
Not sure if we could use different time frames for issues and PRs. Maybe we can see if it makes sense over time and adapt it depending on usage?